### PR TITLE
SSP5_8.5 future compset for watercycle (maint-1.0)

### PR DESCRIPTION
**Pull request by cameronsmith1**
Created new compset for SSP5_8.5 future scenario for CMIP6 watercycle
simulation.  The input files follow the input4MIPS data as closely as
possible.

[BFB, except for SSP585 compset]

### CODE REVIEW

**Comment by cameronsmith1** [comment]
To run the SSP585 configuration using this PR, use the following run_script, available at NERSC:

```
/global/cscratch1/sd/pjc/SSP585_initial_condition/run_e3sm.SSP5_8.5_PR.csh
```

File: components/clm/bld/namelist_files/use_cases/2015-2100_SSP585_transient.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+<!-- <sim_year>1850</sim_year> -->
+
+<sim_year_range>2015-2100</sim_year_range>
+
+<clm_start_type>arb_ic</clm_start_type>
+
+<clm_demand >flanduse_timeseries</clm_demand>
+
+<stream_year_first_ndep phys="clm4_0" bgc="cn"   >1850</stream_year_first_ndep>
```
Comment (bishtgautam) [inline components/clm/bld/namelist_files/use_cases/2015-2100_SSP585_transient.xml]: You can remove all entries for clm4_0 as we don't support it anymore

File: components/clm/bld/namelist_files/use_cases/2015-2100_SSP585_transient.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+<stream_year_first_ndep phys="clm4_0" bgc="cndv" >1850</stream_year_first_ndep>
+<stream_year_last_ndep  phys="clm4_0" bgc="cndv" >2005</stream_year_last_ndep>
+<model_year_align_ndep  phys="clm4_0" bgc="cndv" >1850</model_year_align_ndep>
+
+<stream_year_first_ndep phys="clm4_5" use_cn=".true."   >1850</stream_year_first_ndep>
+<stream_year_last_ndep  phys="clm4_5" use_cn=".true."   >2005</stream_year_last_ndep>
+<model_year_align_ndep  phys="clm4_5" use_cn=".true."   >1850</model_year_align_ndep>
+
+<stream_year_first_ndep phys="clm5_0" use_cn=".true."   >1850</stream_year_first_ndep>
```
Comment (bishtgautam) [inline components/clm/bld/namelist_files/use_cases/2015-2100_SSP585_transient.xml]: Similarly, you can remove all entries for clm5_0 as we don't support it

**Comment by tangq** [comment]
The 5-day test on Edison using the run script (/global/cscratch1/sd/pjc/SSP585_initial_condition/run_e3sm.SSP5_8.5_PR.csh) was successful (reported by @acme-y9s in emails), so I skipped the 5-day test run.

File: components/cam/bld/namelist_files/use_cases/SSP585_cam5_CMIP6.xml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+<prc_exp>3.19D0</prc_exp>
+<prc_exp1>-1.2D0</prc_exp1>
+<se_ftype>2</se_ftype>
+<clubb_C14>1.06D0</clubb_C14>
+<relvar_fix>.true.</relvar_fix>
+<mg_prc_coeff_fix>.true.</mg_prc_coeff_fix>
+<rrtmg_temp_fix>.true.</rrtmg_temp_fix>
+
+<!-- Energy fixer options -->
+<l_ieflx_fix> .true.</l_ieflx_fix>
```
[the remark this answers is not in the document]
Comment (cameronsmith1) [reply components/cam/bld/namelist_files/use_cases/SSP585_cam5_CMIP6.xml]: Hi Qi, I just followed what was on the maint-1.0 branch for the historical compset.   Did the change you describe get put on main-1.0?

**Comment by cameronsmith1** [comment]
Responding to Chris...

> Is using the 1850-2014 average for volcanoes part of the SSP5_8.5 specifications or a choice we made? (It seems like a reasonable option to me).

This was the recommendation of Paul Durack, and there really isn't any other good option.

> My only generic gripe would be about the naming convection and the fact that we are still using 'cam5' in so many places, but that's not something to be fixed by this PR.

Agreed.  However, it is deeply embedded in the logic, so it will require quite a bit of careful work to modify it.

> Regarding the changes suggested by @bishtgautam, we could make them now or part of a subsequent PR.

I suggest we do it in a subsequent PR, just in case there is some unintended consequence (although I don't expect any).

File: components/cam/bld/namelist_files/use_cases/SSP585_cam5_CMIP6.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (tangq) [reply components/cam/bld/namelist_files/use_cases/SSP585_cam5_CMIP6.xml]: That's right. It's on maint-1.0.

[the remark this answers is not in the document]
Comment (acme-y9s) [reply components/cam/bld/namelist_files/use_cases/SSP585_cam5_CMIP6.xml]: I have read through and the changes and done a 5-day test with the branch, all are good to me.

**Comment by cameronsmith1** [comment]
Thank you to all of the reviewers.  Since you have all approved this PR, I propose that we merge this PR to next and test it "as is", since it follows the historical compset.  

However, I agree that it would be good to remove unnecessary and confusing entries.   I am setting up a test to confirm they are BFB with this compset, and then I will submit a new PR.

**Comment by bishtgautam** [comment]
I think this PR will skip next and directly go on maint-1.0. 
cc: @rljacob

**Comment by rljacob** [comment]
Yes that's true.  It goes directly to the maint-1.0 branch.

**Comment by singhbalwinder** [comment]
@bishtgautam : I am assuming that you approve this PR as @cameronsmith1 is going to address your comments in a subsequent PR. I am now going to merge this PR to maint-1.0.

**Comment by cameronsmith1** [comment]
If the PR goes directly on to maint-1.0, then when does it get checked by the testsuite?

**Comment by rljacob** [comment]
The maint-1.0 production test suite runs nightly but it only tests a couple of compsets.

**Comment by cameronsmith1** [comment]
Are those couple of tests the CMIP6 control and historical simulations?  Those are the critical ones for this PR.

BTW, we will also want to merge this PR to master and the other maint-1.x branches.  Do we need separate PRs for those?

**Comment by rljacob** [comment]
Just the pi control and a near-equivalent F-case:
SMS_Ld2.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.edison_intel.allactive-v1cmip6
SMS_Ln5.ne30_ne30.FC5AV1C-L.edison_intel.cam-cosplite

Yes separate PRs are need for the other maint branches.

**Comment by cameronsmith1** [comment]
OK.   I will wait for the nightly testing to go through, and then work on the PRs for master and the other maint branches.

For deleting the unnecessary bits of this PR, do I need to create a new branch?  Or just update the current one I used for this PR?

**Comment by cameronsmith1** [comment]
I checked the tests on CDASH.  I think I found the tests, but it appears they both failed because the account was over its disk quota.

Who is in charge of that account?

**Comment by jgfouca** [comment]
@cameronsmith1 , I am. I'm removing old test results now to get us under the inode quota.

**Comment by cameronsmith1** [comment]
Thanks, Jim!

**Comment by cameronsmith1** [comment]
To implement the additional changes requested by Gautam and Qi, i.e. deleting the unnecessary bits of this PR, do I need to create a new branch? Or just update the current one I used for this PR?

**Comment by bishtgautam** [comment]
I would say create a new branch starting with the latest `maint-1.0`.

**Comment by rljacob** [comment]
Yes new PR from a branch off of maint-1.0.

**Comment by cameronsmith1** [comment]
Hi @jgfouca , CDash say the maint1.0 tests are still failing, but it appears they haven't run since May 5.  Should the tests be working?  How often are they run?

**Comment by ndkeen** [comment]
This morning Jim announced that he stopped testing on edison as it is retiring. I would have kept testing it until the end. :)

**Comment by cameronsmith1** [comment]
I need the tests to confirm that I haven't affected the existing compsets.  Can we reactivate the Edison tests, at least once?  

Are the same tests running on Cori now?

**Comment by jgfouca** [comment]
@cameronsmith1 , they should run tonight.

**Comment by cameronsmith1** [comment]
Thanks, Jim.  I have my fingers crossed for clean results.

**Comment by jgfouca** [comment]
@cameronsmith1 , there was a problem in the Jenkins switch from edison to cori. I have fixed the problem and restarted the test. We should have results in few hours.

**Comment by cameronsmith1** [comment]
Thanks, @jgfouca .  Does this mean the test will run on Cori instead of Edison?  If the test passes, then that will be fine.  If it fails, I will want to rerun the test on Edison to check whether it is my mistake or a machine difference.

**Comment by jgfouca** [comment]
> Does this mean the test will run on Cori instead of Edison?

Yes.

You can always run tests on edison yourself if you need results that aren't on the dashboard. It's as easy as setting your repo to the commit you want to test and:
```
% cime/scripts/create_test e3sm_prod --wait
```

**Comment by cameronsmith1** [comment]
Thanks, @jgfouca .  I ran the test manually.  However, the jobs have been waiting in the queue for several hours.  I included the --wait flag you recommended, but what happens if the window I used dies before the jobs run?

**Comment by jgfouca** [comment]
@cameronsmith1 , as long as the jobs make it to the queue, it shouldn't matter if the connection drops. You'll just have to do a bit of extra work to see what the status of the test results is.

**Comment by cameronsmith1** [comment]
The maint-1.0 tests ran on Edison, and the window stayed up so I could see that it passed.

**Comment by cameronsmith1** [comment]
The maint-1.0 tests also pass on Cori (after the module fix from https://github.com/E3SM-Project/E3SM/pull/2913 )

**Pull request merged**